src/zrc/datamodel/models.py

```diff
@@ -378,6 +399,9 @@ def _get_object(self) -> dict:
                 self._object = client.retrieve(self.object_type.lower(), url=object_url)
         return self._object
 
+    def unique_representation(self):
+        return f"({self.zaak.unique_representation()}) - {self._get_object()['identificatie']}"
```

Comment:
Same deal here with the `identificatie` field - it'll depend on the `object_type`. Maybe use the UUID from the URL for now?

Reply:
changed to uuid from `object` field